R/repo.R

```diff
@@ -43,7 +43,8 @@ repo_auto_link <- function(pkg, text) {
 
   if (!is.null(url$issue)) {
     issue_link <- paste0("<a href='", url$issue, "\\1'>#\\1</a>")
-    text <- gsub("#(\\d+)", issue_link, text, perl = TRUE)
+    text <- gsub("\\(#(\\d+)", paste0("(", issue_link), text, perl = TRUE)
```

Comment:
I think you can do in a single regexp:

```suggestion
    text <- gsub("(\\(|\\s)#(\\d+)", paste0("(", issue_link), text, perl = TRUE)
```